core/src/main/java/co/cask/wrangler/parser/TextDirectives.java

```diff
@@ -62,37 +62,7 @@
 import co.cask.wrangler.steps.row.SetRecordDelimiter;
 import co.cask.wrangler.steps.row.SplitToRows;
 
-import co.cask.wrangler.steps.transformation.CatalogLookup;
-import co.cask.wrangler.steps.transformation.CharacterCut;
-import co.cask.wrangler.steps.transformation.Decode;
-import co.cask.wrangler.steps.transformation.Encode;
-import co.cask.wrangler.steps.transformation.Expression;
-import co.cask.wrangler.steps.transformation.ExtractRegexGroups;
-import co.cask.wrangler.steps.transformation.FillNullOrEmpty;
-import co.cask.wrangler.steps.transformation.FindAndReplace;
-import co.cask.wrangler.steps.transformation.GenerateUUID;
-import co.cask.wrangler.steps.transformation.IndexSplit;
-import co.cask.wrangler.steps.transformation.InvokeHttp;
-import co.cask.wrangler.steps.transformation.Lower;
-import co.cask.wrangler.steps.transformation.MaskNumber;
-import co.cask.wrangler.steps.transformation.MaskShuffle;
-import co.cask.wrangler.steps.transformation.MessageHash;
-import co.cask.wrangler.steps.transformation.Quantization;
-import co.cask.wrangler.steps.transformation.SetColumn;
-import co.cask.wrangler.steps.transformation.Split;
-import co.cask.wrangler.steps.transformation.SplitEmail;
-import co.cask.wrangler.steps.transformation.SplitURL;
-import co.cask.wrangler.steps.transformation.TableLookup;
-import co.cask.wrangler.steps.transformation.TextDistanceMeasure;
-import co.cask.wrangler.steps.transformation.TextMetricMeasure;
-import co.cask.wrangler.steps.transformation.TitleCase;
-import co.cask.wrangler.steps.transformation.Upper;
-import co.cask.wrangler.steps.transformation.UrlEncode;
-import co.cask.wrangler.steps.transformation.XPathArrayElement;
-import co.cask.wrangler.steps.transformation.XPathElement;
-import co.cask.wrangler.steps.transformation.Trim;
-import co.cask.wrangler.steps.transformation.LeftTrim;
-import co.cask.wrangler.steps.transformation.RightTrim;
+import co.cask.wrangler.steps.transformation.*;
```

Comment:
@vhajdari Please follow the coding guideline and setup IntelliJ with the correct profile